File: 1/solutions/code.c
#include <stdio.h>

int main() {
    int upper_limit = 1000;
    int sum = 0;

    for (int i = 1; i < upper_limit; i++) {
        if (i % 3 == 0 || i % 5 == 0) {
            sum += i;
        }
    }
    printf("%d\n", sum);
    return 0;
}

File: 1/solutions/code.go
package main

import "fmt"

func main() {
    upperLimit := 1000

    sum := 0
    for i := 1; i < upperLimit; i++ {
        if i%3 == 0 || i%5 == 0 {
            sum += i
        }
    }
    fmt.Println(sum)
}
